src/sqlfluff/core/parser/grammar/delimited.py

```diff
@@ -212,3 +213,24 @@ def match(
                         return MatchResult(matched_segments.matched_segments, seg_buff)
                     else:
                         return MatchResult.from_unmatched(segments)
+
+
+class CommaDelimited(Delimited):
```

Comment:
IMO it's useful :+1: would be curious on what others think.

Reply:
This approach is ok, but I think it might be even better to make the comma delimited the default for the original `Delimited` segment instead. I completely agree that comma delimited is the most common use, and I think we should just lean into that for the original delimited segment instead. 😄

Reply:
solved. `CommaSegment` is now the default delimiter in `Delimited`. `CommaDelimited` was removed.